### PR TITLE
[FSSDK-8955] Refac: Replaced all instances of full stack  from code comments and in nuspec except owner.

### PR DESCRIPTION
**Pull request by mnoman09**
## Summary
- Replaced all instances of full stack  from code comments and in nuspec except owner.
- This PR does not contain changes in md files.

## Test plan
All tests should pass. 

## Issues
- ["FSSDK-8955"](https://jira.sso.episerver.net/browse/FSSDK-8955)

### CODE REVIEW

File: OptimizelySDK.Package/OptimizelySDK.nuspec [hunk 1]
```diff
@@ -11,7 +11,7 @@
         <icon>OptimizelySDK.png</icon>
         <iconUrl>https://github.com/optimizely/csharp-sdk/blob/master/OptimizelySDK.png?raw=true</iconUrl>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <description>C# SDK for Optimizely X Fullstack</description>
+        <description>C# SDK for Optimizely Feature Experimentation</description>
```
Comment (russell-loube-optimizely) [inline OptimizelySDK.Package/OptimizelySDK.nuspec:14]:
```suggestion
        <description>C# SDK for Optimizely Feature Experimentation, Optimizely Full Stack (legacy), and Optimizely Rollouts</description>
```

Comment (mikechu-optimizely) [reply OptimizelySDK.Package/OptimizelySDK.nuspec:14]: I agree with this commit suggestion.

**Review by mikechu-optimizely — CHANGES_REQUESTED** [review]

**Comment by mikechu-optimizely** [comment]
@mnoman09 Should we combine with #331 so we have 1 squashed commit to cherry pick?

**Comment by mnoman09** [comment]
> @mnoman09 Should we combine with #331 so we have 1 squashed commit to cherry pick?

No merge them separately, so that when we cherry pick it would be easier.

**Review by jaeopt — APPROVED** [review]
LGTM

**Review by russell-loube-optimizely — COMMENTED** [review]
LGTM

**Review by mikechu-optimizely — APPROVED** [review]
👀✅2️⃣🤓

**Pull request merged**